README.md

```diff
@@ -8,7 +8,7 @@ You may implement it using either JavaScript or preferably TypeScript, and are w
 
 ## Overview
 
-Implement the requirements described below and push your solution to GitHub for discussion in the scheduled technical interview. You are welcome to use a private repository as long as it is visible to `@mkerollos` `@danrauv` `@cgcurta` `@LWilliams-Annaise` and `@noejon`.
+Implement the requirements described below and push your solution to GitHub for discussion in the scheduled technical interview. You are welcome to use a private repository as long as it is visible to `@mkerollos` `@danrauv` `@cgcurta` `@LWilliams-Annaise` `@lucitemple` `@nich1jb` `@RicoJap` and `@noejon`.
```

Comment:
I would remove Mark Danny and Cal. It is daunting if there are too many people on there (there are already a lot just with the developers)